proto/rill/runtime/v1/queries.proto

```diff
@@ -511,6 +518,21 @@ message MetricsViewTimeRangeResponse {
   TimeRangeSummary time_range_summary = 1;
 }
 
+message MetricsViewDataTypesRequest {
+  string instance_id = 1;
+  string metrics_view_name = 2 [(validate.rules).string.min_len = 1];
+  int32 priority = 3;
+}
+
+message MetricsViewDataTypesResponse {
+  repeated MetricsViewDataType data_types = 1;
+}
+
+message MetricsViewDataType {
+  string name = 1;
+  string type = 2;
+}
```

Comment:
It would be preferable to return a `rill.runtime.v1.StructType`, which is our generic type for representing schemas (e.g. see `MetricsViewAggregationResponse`). Then it will work predictably for both DuckDB and Druid (and any other OLAP backends we add in future).